src/main/java/com/laserfiche/api/client/APIServer/TokenClient.java

```diff
@@ -0,0 +1,16 @@
+package com.laserfiche.api.client.APIServer;
```

Comment:
Would it make more sense to call this something like `package com.laserfiche.api.client.selfhosted;`?

Also, Java package name should be all lower case.

Reply:
done